cmd/root/keymap.go

```diff
@@ -115,3 +118,187 @@ func (k keymap) render(char string) (km string) {
 	}
 	return
 }
+
+const (
+	lpinky uint = iota
+	lring
+	lmiddle
+	lindex
+	lthumb
+	rthumb
+	rindex
+	rmiddle
+	rring
+	rpinky
+)
+
+// Rune to fingers
+var rtfs = map[rune][]uint{
```

Comment:
This is also dependent on the keymap. The fingers that are used are dependent on the location of the keys, so if they keys change location, then the fingers used to press those keys changes.

However, [I would like this to be configurable](https://github.com/NicksPatties/sweet/issues/53), since I cannot expect people to have the same keyboard layout